scripts/generate_simple_digest.py

```
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
```

Ruff (0.14.3)

Shebang is present but file is not executable

(EXE001)